test/test_core.py

```diff
@@ -5859,6 +5859,8 @@ def test_fs_rename_on_existing(self):
     self.do_runf('fs/test_fs_rename_on_existing.c', 'success')
 
   @also_with_nodefs_both
+  @crossplatform
```

Comment:
What does `@crossplatform` do? I had assumed that it wouldn't run on windows if we didn't include `@crossplatform`. Does it add crossplatform tests to the PR CI? And they are always run on the main branch?

Reply:
It's slightly subtle. the actual `@crossplatform` decorator just marks the test as being a test that might have different behavior across platforms, and we run those tests on Mac and Windows on the CI that runs on Github (we run only those test to keep CI times costs down). But the builders we use to build the EMSDK releases (which run on [chromium's](https://ci.chromium.org/p/emscripten-releases/g/main/console?limit=200) CI infrastructure) do run all of the 'other' and some of the core test suites on Mac and Windows. So that's why we've been catching these failures after they've already landed in the emscripten repo. Usually it works fine, but you've hit basically the one area of work where it's an annoyance. :) Probably for these kinds of tests you should either mark them `@crossplatform` or explicitly disable them on Windows. As for how much effort you should put in, noderawfs has always sort of been "best-effort" on Windows. Maybe we should try to come up with a better-defined policy, but maybe for now a decent tradeoff would be to start by trying out your tests as crossplatform and if it's not too much effort to make them work, do that, and otherwise just disable on Windows?